README.md

```diff
@@ -64,8 +71,10 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
-```
+function doSomething(thingToDo) {
+  thingToDo.preventDefault
+})
+$('doSomething')
```

Comment:
This question doesn't require jQuery or preventDefault. (Please see the answer key when we share it for more info)